timepiece/entries/models.py

```diff
@@ -355,6 +357,7 @@ def clean(self):
         )
         if (entries.exists() and not self.id
                 or self.id and self.status == Entry.INVOICED):
+            # if not user.is_superuser:
```

Comment:
I am not familiar with this code, but is this comment accurate?  How does being a superuser short circuit this if block?